CanDIG/CanDIGv2
# Tyk add new API feature + New API for Katsu/Chord Metadata

Two new things added in this PR - 

1. A simple command, to redeploy Tyk after addition of new API
2. Katsu API added at the location `/katsu`

To test this, 
- deploy the CanDIGv2 stack
- run `make init-authx` 
You should see Katsu API under `http://<host:port>/katsu`



DIG-653
DIG-652
